.github/actions/test/integration/test/cloudhypervisor/vm.xml

```diff
@@ -23,15 +23,13 @@
             <source network='default' />
             <model type='virtio' />
         </interface>
-        <!-- currently not supported with libvirt/chv: -->
-        <!-- error: internal error: Serial only works in UNIX/PTY modes -->
-        <!-- <console type='pty'> -->
-        <!--     <target type='serial' port='0' /> -->
-        <!-- </console> -->
-        <!-- <serial type='file'> -->
-        <!--     <source path='/var/log/CHV.log' /> -->
-        <!--     <target port='0' /> -->
-        <!-- </serial> -->
+        <console type='pty'>
+            <target type='serial' port='0' />
+        </console>
+        <serial type='file'>
+            <source path='/var/log/CHV.log' />
+            <target port='0' />
+        </serial>
```

Comment:
I remember the current libvirt chv had some limitations about number of serials or consoles being limited to a single one. Have you checked if configuring console + serial this way works with recent libvirt/chv?

Reply:
(having said that - the VM on HV starts up, serial log is produced on HV1... but the migration fails)